src/components/Request/FormFields/LinkedinSearchSkills.tsx

```diff
@@ -0,0 +1,125 @@
+import { InfoLabel, Input, Text } from "@fluentui/react-components";
+import { Controller } from "react-hook-form";
+import { FormField } from "components/Request/NewRequestForm";
+import "components/Request/Request.css";
+import { TextFieldIcon } from "@fluentui/react-icons-mdl2";
+
+const rules = {
+  maxLength: {
+    value: 255,
+    message: "Skills can be no longer than 255 characters",
+  },
+};
+
+const LinkedinSearchSkills = ({ name, form }: FormField) => {
+  return (
+    <fieldset>
+      <legend>
+        <TextFieldIcon className="requestFieldIcon" />
+        <InfoLabel weight="semibold" info="C++, MATLAB, etc.">
+          Skills to Search For
```

Comment:
Maybe stay consistent with the others and drop "to Search For" as these are all criteria to search for
```suggestion
          Skills
```